### PR TITLE
fix: FEND-010 add contact page

### PR DESCRIPTION
**Pull request by AHoncharyk**

### CODE REVIEW

File: src/pages/Contact.jsx [hunk 1]
```diff
@@ -1,11 +1,144 @@
...
+          selected={selectedDateTime}
+          onChange={(date) => setSelectedDateTime(date)}
+          showTimeSelect
+          dateFormat="Pp"
+          filterTime={(time) => {
+            const hours = time.getHours();
+            return hours >= 10 && hours < 24;
+          }}
+          minDate={new Date()}
+          maxDate={new Date(new Date().setMonth(new Date().getMonth() + 1))}
```
Comment (KaterynaBabii) [inline src/pages/Contact.jsx]: please move to constants before render

File: src/pages/Contact.jsx [hunk 2]
```diff
@@ -1,11 +1,144 @@
...
+          className={`form-input ${errors.people ? "error-border" : ""}`}
+          placeholder="How many people"
+          value={formData.people}
+          onChange={handleChange}
+        />
+        {errors.people && <p className="error-text">{errors.people}</p>}
+        
+        <DatePicker
+          selected={selectedDateTime}
+          onChange={(date) => setSelectedDateTime(date)}
```
Comment (KaterynaBabii) [inline src/pages/Contact.jsx]: please to move to function

File: package.json [hunk 3]
```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "date-fns": "^4.1.0",
```
Comment (KaterynaBabii) [inline package.json]: Are you using date-fns anywhere?

File: src/pages/Contact.jsx [hunk 4]
```diff
@@ -1,11 +1,144 @@
...
+        
+        <textarea
+          name="message"
+          className={`form-input ${errors.message ? "error-border" : ""}`}
+          placeholder="Message / Special Requirements"
+          value={formData.message}
+          onChange={handleChange}
+          maxLength={250}
+        />
+        {errors.message && <p className="error-text">{errors.message}</p>}
```
Comment (KaterynaBabii) [inline src/pages/Contact.jsx]: better use errors?.message to avoid errors

File: src/pages/Contact.jsx (hunk 1 above)
Comment (AHoncharyk) [reply src/pages/Contact.jsx]: fixed

File: src/pages/Contact.jsx (hunk 4 above)
Comment (AHoncharyk) [reply src/pages/Contact.jsx]: fixed

File: package.json (hunk 3 above)
Comment (AHoncharyk) [reply package.json]: fixed

File: src/pages/Contact.jsx (hunk 2 above)
Comment (AHoncharyk) [reply src/pages/Contact.jsx]: fixed

**Review by KaterynaBabii — APPROVED** [review]

**Pull request merged**